### PR TITLE
bugfix: arbitrary values and route matching

### PR DESCRIPTION
**Pull request by k10wl**

### CODE REVIEW

File: components/molecules/chat/preview.vue [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+        </p>
+      </AtomsCard>
+    </NuxtLink>
+
+    <button
+      type="button"
+      class="absolute top-1/2 right-5 flex h-4 w-4 shrink-0 -translate-y-1/2 items-center"
+      @click="($event) => $event.stopPropagation()"
+    >
+      <IconsClose class="fill-gray-400 hover:fill-gray-200" />
```
Comment (azhechkova) [inline components/molecules/chat/preview.vue:25]: maybe it's better to remove the button from dom

File: components/molecules/chanel/navigation/base.vue [hunk 2]
```diff
@@ -1,18 +1,10 @@
 <template>
-  <NuxtLink class="group">
-    <AtomsCenteredSquare
-      class="rounded-[50%] bg-gray-800 text-gray-300 transition-all ease-in-out active:translate-y-0.5 group-hover:rounded-2xl group-hover:bg-blue-600 aria-selected:rounded-2xl aria-selected:bg-blue-600"
-      :aria-selected="selected"
-    >
+  <NuxtLink
+    class="group overflow-hidden rounded-[50%] bg-gray-800 transition-all ease-in-out hover:rounded-2xl hover:bg-blue-600 active:translate-y-0.5"
+    active-class="!rounded-2xl !bg-blue-600"
```
Comment (azhechkova) [inline components/molecules/chanel/navigation/base.vue:4]: why do we need '!' before the class name here?)

File: components/molecules/chat/preview.vue (hunk 1 above)
Comment (k10wl) [reply components/molecules/chat/preview.vue:25]: a11y reasons. let it be, its rent-free

File: components/molecules/chanel/navigation/base.vue (hunk 2 above)
Comment (k10wl) [reply components/molecules/chanel/navigation/base.vue:4]: good question. it does not work without it! :D 

I believe that the reason is cascading. feels like in tw `bg-gray-800` is declared after `bg-gray-600` and therefore 600 is overridden by 800.

**Review by azhechkova — APPROVED** [review]

File: components/molecules/chanel/navigation/base.vue (hunk 2 above)
Comment (azhechkova) [reply components/molecules/chanel/navigation/base.vue:4]: 👍🏻

**Pull request merged**